docs/faq.md

```diff
@@ -77,3 +77,7 @@ If you want to use Northstar you'll have to purchase Titanfall 2 either from Ste
 ### Q: Where is Frontier Defense?
 
 A: Frontier Defense is currently in development and will be available in a release when it's finished.
+
+### Q: Why is my game running so bad?
+
+A: This usually lies behind your computer's specs not being high enough in order to run the game, but can also be affected by EA App's overlay. You can disable this by going to the EA App, clicking on your profile, clicking on the drop down arrow by your username, going to settings, going to `Application`, and scrolling down to "In Game Overlay", and changing it to be off.
```

Comment:
Some screenshots for this would be nice, though if not, a text without would be fine for now ^^

Reply:
Not home and this laptop doesn't have ea working, but do we want images in the FAQ section? Would hate to accidentally take up too much space or anything, though if we're asking users to `ctrl + f` search, this isn't very relevant

Reply:
Fair point, they should be in trouble shooting section then. But that actually raises a good point. Maybe the text telling one how to disable EA ap should be in troubleshooting with a link to it in FAQ?